event/src/main/resources/application.conf

```diff
@@ -0,0 +1,37 @@
+// Important: enable HTTP/2 in ActorSystem's config
+akka.http.server.preview.enable-http2 = on
+
+akka.actor.provider = cluster
+
+//FIXME - autocreate only for dev
+akka.actor.serialization-bindings{
+  "com.improving.app.common.serialize.PBMsgSerializable" = proto
+}
+
+akka.remote.artery.canonical.port = 0
+
+akka.persistence {
+  journal.plugin = "akka.persistence.cassandra.journal"
+  snapshot-store.plugin = "akka.persistence.cassandra.snapshot"
+
+  cassandra {
```

Comment:
cassandra and not scylla? or am I missing something

Reply:
Cassandra connector is needed for scylla operation